docs/Contributing/Run-Locally-Built-Fleetd.md

```diff
@@ -1,16 +1,16 @@
-# Run Locally Built Orbit 
+# Run Locally Built Fleetd 
 (MacOS)
 
 
-### Run fleet server (And the released Orbit).
-In order to run a local agent (Orbit + osquery) the first step is to run the fleet server locally.
+### Run fleet server (And the released Fleetd).
+In order to run a local agent (Fleetd + osquery) the first step is to run the Fleet server locally.
 Follow this document which will run it together with the released agent.
 https://fleetdm.com/docs/contributing/building-fleet
 
-### Modify the Orbit code as needed
+### Modify the Fleetd code as needed
 
 ### Build and run locally.
-In order to use a local version we need to create a local TUF service that will point the installer to take the local Orbit (Instead of the official one)
+In order to use a local version we need to create a local TUF service that will point the installer to take the local Fleetd (Instead of the official one)
```

Comment:
Another sneaky capital letter, may as well nab them as we see them:
```suggestion
In order to use a local version we need to create a local TUF service that will point the installer to take the local Fleetd (instead of the official one).
```